docs/htdocs/info/docs/tools/vep/vep_formats.html

```diff
@@ -577,6 +586,17 @@ <h2 id="vcfout"> VCF output</h2>
     Here is a list of the (default) fields you can find within the CSQ field:
     <pre class="code sh_sh">Allele|Consequence|IMPACT|SYMBOL|Gene|Feature_type|Feature|BIOTYPE|EXON|INTRON|HGVSc|HGVSp|cDNA_position|CDS_position|Protein_position|Amino_acids|Codons|Existing_variation|DISTANCE|STRAND|FLAGS|SYMBOL_SOURCE|HGNC_ID</pre>
     </p>
+
+    <div>
+      <div class="info" style="float:left">
+        <h3>Note</h3>
+        <div class="message-pad">
+          <p>The Existing_variation column is only populated if we use option <a href="script/vep_options.html#opt_check_existing">--check_existing</a> or any other option that switches it on (--af_1kg silently switches on --check_existing).</p>
```

Comment:
Other fields are not reported by default and need configuring including HGVS, biotype and symbol. 
Would it be clearest to list those that are reported by default and say the rest mut be configured?

Reply:
There are more fields that are reported than fields not being reported. I think biotype is reported by default, HGVS, Symbol and Existing_variation need to be configured. We could tell these 3 fields are not reported by default and need configuring.